lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Model for Course Programs.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.context = data;
+                    //we should populate our model by looking at the run_modes
+                    if (data.run_modes.length > 0){
+                        //We only have 1 run mode for this program
+                        this.setActiveRunMode(data.run_modes[0]);
+                    }
+                }
+            },
+
+            setActiveRunMode: function(runMode){
+                this.set({
+                    display_name: this.context.display_name,
+                    marketing_url: this.context.marketing_url || '',
+                    key: this.context.key,
+                    course_start: runMode.course_start,
```

Comment:
These field names aren't consistent with the ones proposed at https://gist.github.com/rlucioni/e66e23a1efe83b7956e08d1b82f7d9bf.